modules/configuring-a-pipeline-server.adoc

```diff
@@ -44,7 +44,8 @@ ifdef::upstream[]
 endif::[]
 * You have created a data science project that you can add a pipeline server to.
 * You have an existing S3-compatible object storage bucket and you have configured write access to your S3 bucket on your storage account.
-* If you are configuring a pipeline server with an external MySQL database, your database must use MySQL version 5.x.
+* If you are configuring a pipeline server with an external MySQL database, your database must use at least MySQL version 5.x. However, RedHat recommends that you use MySQL version 8.x. 
```

Comment:
Space needed in 'RedHat', but can sub in the org-name variable for this line and line 48.

Reply:
Good spot, changed both lines.